validator/accounts/v2/accounts_exit.go

```diff
@@ -17,19 +16,20 @@ import (
 
 // ExitAccounts performs a voluntary exit on one or more accounts.
 func ExitAccounts(cliCtx *cli.Context, r io.Reader) error {
```

Comment:
This function should be named `ExitAccountsCLI`.

By the way, I should add the CLI-decoupling to voluntary exits, right?

Reply:
Im not sure if we should. Exits are so scary and perhaps we want to enforce CLI only? Something to think about